src_c/math.c

```diff
@@ -1287,6 +1293,95 @@ vector_scale_to_length(pgVector *self, PyObject *length)
     Py_RETURN_NONE;
 }
 
+static int
+_vector_move_towards_helper(Py_ssize_t dim, double *origin_coords, double *target_coords, double distance)
```

Comment:
perhaps this can become a `static void` function. Because all codepaths seem to return `1`, so it is redundant to be checking function output here